src/ChainSafe.Gaming.SygmaClient/Contracts/BasicFeeHandler.cs

```diff
@@ -19,22 +20,23 @@ public class BasicFeeHandler : IBasicHandler
         public BasicFeeHandler(IContractBuilder cb, string address)
         {
             this.address = address;
-            this.contract = cb.Build(FeeHandlerAbi, address);
+            contract = cb.Build(FeeHandlerAbi, address);
```

Comment:
The FeeHandlerAbi, you can replace the huge constant declared inside the file with an embedded resource. There is an example in the Marketplace project.
There is also a helper method to read ABI from resources. It's called `AbiHelper.ReadAbiFromResources`.